Hello, traveler. This is the beginning of a log file called: "Fascist Tweet Archiver".
Some basic system info about the host machine: posix.uname_result(sysname='Linux', nodename='file-server', release='3.16.0-6-686-pae', version='#1 SMP Debian 3.16.57-2 (2018-07-14)', machine='i686')
This log was begun Friday, 24 August 2018 at 00:13



INFO: read target account data; we're tracking 59 accounts
WARNING: no ProtocolError (got exception "name 'ProtocolError' is not defined");
 trying requests.packages.urllib3.exceptions instead
NOTE: successfully imported from requests
